Babylon/groups/api/groups/workspace/commands/get.py

```diff
@@ -0,0 +1,118 @@
+import json
+from logging import getLogger
+from pprint import pformat
+from typing import Optional
+
+from click import Path
+from click import argument
+from click import command
+from click import make_pass_decorator
+from click import option
+from cosmotech_api.api.workspace_api import WorkspaceApi
+from cosmotech_api.exceptions import NotFoundException
+from cosmotech_api.exceptions import UnauthorizedException
+
+from ......utils.api import convert_keys_case
+from ......utils.api import filter_api_response_item
+from ......utils.api import get_api_file
+from ......utils.api import underscore_to_camel
+from ......utils.decorators import allow_dry_run
+from ......utils.decorators import require_deployment_key
+from ......utils.decorators import timing_decorator
+
+logger = getLogger("Babylon")
+
+pass_workspace_api = make_pass_decorator(WorkspaceApi)
+
+
+@command()
+@allow_dry_run
+@pass_workspace_api
+@timing_decorator
+@argument("workspace_id", type=str)
+@require_deployment_key("organization_id", "organization_id")
+@option(
+    "-o",
+    "--output-file",
+    "output_file",
+    help="File to which content should be outputted (json-formatted)",
+    type=Path(),
+)
+@option(
+    "-f",
+    "--fields",
+    "fields",
+    type=str,
+    help="Fields witch will be keep in response data, by default all",
+)
+@option(
+    "-e",
+    "--use-working-dir-file",
+    "use_working_dir_file",
+    is_flag=True,
+    help="Should the path be relative to the working directory ?",
+    type=bool,
+)
+@option(
+    "--from-file",
+    "from_file",
+    is_flag=True,
+    help="In case the workspace id is retrieved from a file",
+)
```

Comment:
Don't use from file, prefer a `-f` option that will take a path
And set required=False on the argument and check if you have one of them